crates/biome_js_analyze/src/analyzers/nursery/no_useless_ternary.rs

```diff
@@ -0,0 +1,94 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_js_syntax::{AnyJsExpression, JsConditionalExpression, JsSyntaxKind};
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Disallow ternary operators when simpler alternatives exist
+    ///
+    /// It’s a common mistake in JavaScript to use a conditional expression to select between two
+    /// Boolean values instead of using ! to convert the test to a Boolean.
```

Comment:
```suggestion
    /// It’s a common mistake in JavaScript to use a conditional expression to select between two
    /// boolean values instead of using the logical NOT (`!`) to convert the test to a boolean.
```